Source/module.modulemap

```diff
@@ -0,0 +1,29 @@
+module OCMock {
+  umbrella header "OCMock/OCMock.h"
+  exclude header "OCMock/NSInvocation+OCMAdditions.h"
```

Comment:
rather than having to include every existing .h as well as having to change this modulemap (and thus, technically, a small aspect of the API) every time the internal structure of OCMock changes in a way that causes the addition or deletion of headers, i would make this module.modulemap simply refer specifically to each header to be included.

that way, only when public headers get added or removed will this file also change … and the compiler will pretty quickly enforce such changes in the first thing that attempts to import the module.

possibly reference:

https://github.com/twitter/ios-twitter-logging-service/blob/master/Resources/module.modulemap

Reply:
Commit coming that replaces the umbrella and excludes with a list of the public headers. Thanks!